src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_ELEMENT_MAX_SIZE = 100;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        return new Circle(Color.WHITE.name(), 10);
+    }
+
+    public Figure getRandomFigure() {
+        switch (random.nextInt(5)) {
+            case 0:
+                return new Square(colorSupplier.getRandomColor(),
+                        random.nextInt(FIGURE_ELEMENT_MAX_SIZE));
```

Comment:
lets add private methods to get each random figure - this would make your code easier to read
```suggestion
            case 0:
                return getRandomSquare();
```